README.md

```diff
@@ -13,76 +13,33 @@ Once created, this project can be easily and quickly deployed to Vercel from the
 
 First, install the dependencies by running one of these commands, depending on your current setup:
 ```bash
-npm install
-# or
-yarn install
+npm install # or yarn install
 ```
-After running the command, you should see a message:
-```
-ready - started server on 0.0.0.0:3000, url: http://localhost:3000
-```
-Congratulations! Your starter project is now live and can be viewed at [http://localhost:3000](http://localhost:3000).
-
-If you use yarn, you might want to delete `package-lock.json` file to avoid possible  resolution inconsistencies caused by unsynchronized lock files.
 
 ## 2) Set API Token
 
-For Next.js to be able to fetch your ButterCMS content properly, you need to add your API token as an environment variable. To do that, copy `env.sample` and edit it, so that includes your API token.
+To fetch your ButterCMS content, add your API token as an environment variable.
 
 ```bash
-$ cp env.sample .env
-```
-```
-# .env
-NEXT_PUBLIC_BUTTER_CMS_API_KEY=<Your API Token>
+$ echo 'NEXT_PUBLIC_BUTTER_CMS_API_KEY=<Your API Token>' >> .env
 ```
 
 ## 3) Run the local server
-You'll want to run server any time you want to view the project locally:
 
-```bash
-npm run dev # or yarn dev
-```
-
-To quit the server press CONTROL-C.
-
-### Build the app
-Use one of these commands which build the application for production usage:
+To view the app in the browser, you'll need to run the local development server:
```

Comment:
```suggestion
To view the app in a browser, you'll need to run the local development server:
```
If you're not referring to a specific browser, I believe the indefinite article is likely correct.